swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, id=None, condition= 0):
+        # self.id = id if id is not None else uuid.uuid4().int
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+
+    def get_category(self):
+        return "Item"
+    
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
+    
+
+    def condition_description(self):
+        if self.condition in range (0,1):
+            return "horrible"
+        elif self.condition in range (1,2):
+            return "poor"
+        elif self.condition in range (2,3):
+            return "fair"
+        elif self.condition in range (3,4):
+            return "good"
+        else:
+            return "mint"
```

Comment:
This set of conditionals does what we need it to do, but it does start to get a bit cumbersome. Might there be a way for us to use a different structure that can accomplish the same thing?

With that in mind, one of the benefits of the conditionals is that we can play with ranges pretty easily, which is amazing! If we used something with a different structure however, we could absolutely use a function that may be able to round up or round down decimal points!